ceph-releases/jewel/ubuntu/14.04/daemon/config.k8s.sh

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
 set -e
 
+# log arguments with timestamp
+function log {
```

Comment:
+1 it'd be nice to have more granularity later. Ideally things like function name being executed etc. We can start slowly with this PR.